docs/source/conf.py

```diff
@@ -79,7 +79,7 @@
 napoleon_use_admonition_for_references = False
 napoleon_use_ivar = False
 napoleon_use_param = True
-napoleon_use_rtype = True
+napoleon_use_rtype = False
```

Comment:
This example code:
```python
# cleanlab/internal/token_classification_utils.py
def get_sentence(words: List[str]) -> str:
    """
    Get sentence formed by a list of words with minor processing for readability

    Parameters
    ----------
    words:
        list of word-level tokens

    Returns
    ----------
    sentence formed by list of word-level tokens

    Examples
    ----------
    >>> from cleanlab.internal.token_classification_utils import get_sentence
    >>> words = ["This", "is", "a", "sentence", "."]
    >>> get_sentence(words)
    'This is a sentence.'
    """
    ...
```

would output the following in the docs.

![image](https://user-images.githubusercontent.com/18127060/189932307-6e3e7610-2529-43f9-91f5-df3e9994e513.png)

Reply:
I don't see a clean way to handle the ` `` ` around the description of the return value.